### PR TITLE
docs: add evdev documentation

### PR DESCRIPTION
**Pull request by paulhamsh**
docs: documentation for evdev

### Description of the feature or fix

New documentation for Linux evdev in LVGL v9

### CODE REVIEW

File: docs/integration/driver/touchpad/evdev.rst [hunk 1]
```diff
@@ -0,0 +1,57 @@
+========================
```
Comment (FASTSHIFT) [inline docs/integration/driver/touchpad/evdev.rst]: Align delimiters with word length

File: docs/integration/driver/touchpad/evdev.rst [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+Linux Evdev Driver
+========================
```
Comment (kisvegabor) [inline docs/integration/driver/touchpad/evdev.rst]:
```suggestion
Linux Evdev Driver
==================
```

File: docs/integration/driver/touchpad/evdev.rst (hunk 1 above)
Comment (kisvegabor) [inline docs/integration/driver/touchpad/evdev.rst]:
```suggestion
==================
```

**Review by kisvegabor — APPROVED** [review]
Looks good, thank you! 

@FASTSHIFT I've fixed the formatting issue.

**Review by FASTSHIFT — APPROVED** [review]

**Pull request merged**